samples/azure-quantum/entanglement-swap/ES-quantinuum-qiskit.ipynb

```diff
@@ -0,0 +1,930 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "all_results = dict()\n",
+    "\n",
+    "def get_fidelity(counts):\n",
+    "    total = sum(counts.values())\n",
+    "    return (counts['1 1'] + counts ['0 0']) / total * 100\n",
+    "\n",
+    "for i in range(4,20):\n",
```

Comment:
[pep8 nit]
```suggestion
    "for i in range(4, 20):\n",
```
Even though this is technically correct, I might suggest changing to a different number to avoid even accidental 420 references.